app/src/main/java/com/phantom/onetapfacebookmodule/FacebookHook.java

```diff
@@ -38,16 +38,18 @@ protected void afterHookedMethod(XC_MethodHook.MethodHookParam hookParams) throw
                 try {
                     Uri uri = (Uri) hookParams.args[0];
                     ApplicationLogMaintainer.sendBroadcast(context, "Facebook Main URL : " + uri.toString());
-                    String stringUri = "";
```

Comment:
Well, it would be better to just replace whole patch with this line :

`String stringUrl = uri.toString();`

Anyway, Thanks a lot of contributing!
If you ain't free, I can change it for you :)